.github/workflows/commitlint.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: Commitlint
+on:
+  push:
+    branches-ignore:
+      - "main"
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 5
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: Fetch main
+        run: git fetch --no-tags --depth=50 origin main
+
+      - run: make install
+      - name: Commitlint
```

Comment:
(Maybe a blank line here…?)

Reply:
Added blank line and `name` for install section.